borg/remote.py

```diff
@@ -308,7 +314,7 @@ def handle_error(error, res):
             elif error == b'ObjectNotFound':
                 raise Repository.ObjectNotFound(res[0], self.location.orig)
             elif error == b'InvalidRPCMethod':
-                raise InvalidRPCMethod(*res)
+                raise InvalidRPCMethod(res[0].decode())
```

Comment:
"Invalid RPC method 82" (unpacking bytes = ints) => "RPC method negotiatea is not valid"